templates/shared/_client-contact-us-form.html

```diff
@@ -1,13 +1,16 @@
 <section class="p-strip is-deep">
   <div class="row">
     <div class="col-8">
-      <h1>{{h1}}</h1>
-      <p>{{intro_text}}</p>
+      <h1>{{ h1 }}</h1>
```

Comment:
Also ideally, we shouldn't name things the same as HTML tags or programming language keywords, so we don't mistakengly override things. In this case it should be fine, it's HTML, but it's a good habit to not name it this way.